app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Api
+  module V1
+    class CommentsController < ApplicationController
+      before_action :set_comment, only: %i[update destroy]
+
+      def create
+        @comment = @commentable.comments.build(comment_params)
+        @comment.user = current_user
```

Comment:
What happens if I am not logged in to the system?

Reply:
Later, an unregistered user will not have access to these actions